### ISSUE TITLE
Documentation on how to bridge rooms

### ISSUE THREAD

**Issue by turt2live** [exul/matrix-rocketchat#67]
As far as I can tell, there's no documentation to explain how to bridge to Rocket.Chat. Some information as to how to register the application service, and how to set up bridges, would be awesome :)

**Comment by exul** [comment]
You're right, there isn't much yet. The sample config file (https://github.com/exul/matrix-rocketchat/blob/master/config/config.yml.sample) contains some comments regarding the setup. The matrix/synapse side works in the same way as it does for other bridges.
After the application service is registered, a 1:1 chat can be started with the bot user to get further instructions.
I'll put together a doc to describe the setup process (with a big disclaimer that nobody has run the bridge in production yet and nasty things could happen ;-)).

**Comment by turt2live** [comment]
How do you use the bridge after starting it though? I can't find anything that says "invite this user" or "join this room" or something.

**Comment by exul** [comment]
The instructions to compile and start the bridge are in the readme: https://github.com/exul/matrix-rocketchat#compiling-from-source

If it doesn't work I'd consider that a bug, so if it doesn't work for you, just let me know.

**Comment by MurzNN** [comment]
I have found Docker image https://hub.docker.com/r/exul/matrix-rocketchat-dev - maybe it will easier for setup, that manual compiling?

**Comment by MurzNN** [comment]
@turt2live Did you have any progress with testing? Does bot instance created and answer to commands?

**Comment by turt2live** [comment]
I haven't had a chance yet. From what I understand though, it's fairly straightforward once you start a chat with the bot.

**Comment by exul** [comment]
There is a draft of a document that describes the setup here: https://github.com/exul/matrix-rocketchat/blob/setup-docs/doc/setup.md
But it's based on a binary release that doesn't exist yet. Before I provide a binary (even if it's not 1.0), I want to change the way auth tokens are handled and I did not yet have the time to finish that.
And as @turt2live said, once you've started the chat with the bot, it should be straightforward (the bot will give you instructions).
